moonworm/crawler/networks.py

```diff
@@ -13,39 +13,17 @@
         PolygonBlock,
         PolygonLabel,
         PolygonTransaction,
+        WyrmBlock,
+        WyrmLabel,
+        WyrmTransaction,
         XDaiBlock,
         XDaiLabel,
         XDaiTransaction,
     )
+    from moonstreamdb.moonworm import Network, MODELS, tx_raw_types
```

Comment:
It seems a little confusing for me to call the module `moonstreamdb.moonworm`. But I don't have a better name for it.

Reply:
networks?